test/src/test/java/org/corfudb/infrastructure/TestLayoutBuilder.java

```diff
@@ -49,7 +50,7 @@ public static Layout single(int port) {
                 .addLogUnit(port)
                 .addToSegment()
                 .addToLayout()
-                .setClusterId(UUID.randomUUID())
+                .setClusterId(UUID.fromString("00000000-0000-0000-0000-000000000000"))
```

Comment:
You can use Layout.INVALID_CLUSTER_ID, please replace occurences in other test files.

Reply:
Fixed